docs/user_guide/assessment.rst

```diff
@@ -1,20 +1,134 @@
 Assessment
 ==========
 
-Metrics
--------
+Introduction
+------------
+
+The goal of fairness assessment is to answer the question: Which groups of 
+people may be disproportionately negatively impacted by an AI system and in 
+what ways?
+
+The steps of the assesment are as follows:
+1. Identify types of harms
+2. Identify the groups that might be harmed
+3. Quantify harms
+4. Compare quantified harms across the groups
+
+We next examine these four steps in more detail.
+
+Identify types of harms
+^^^^^^^^^^^^^^^^^^^^^^^
+
+See :ref:`types_of_harms` for a guide to types of fairness-related harms. 
+For example, in a system for screening job applications, qualified candidates 
+that are automatically rejected experience an allocation harm. In a 
+speech-to-text transcription system, disparities in word error rates for 
+different groups may result in harms due to differences in the quality of service.
+Note that one system can lead to multiple harms, and different types of 
+harms are not mutually exclusive. For more information, review 
+Fairlearn's `2021 SciPy tutorial <https://github.com/fairlearn/talks/blob/main/2021_scipy_tutorial/overview.pdf>`_.
+
+Identify the groups that might be harmed
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In most applications, we consider demographic groups including historically 
+marginalized groups (e.g., based on gender, race, ethnicity). We should also 
+consider groups that are relevant to a particular use case or deployment context. For example, for 
+speech-to-text transcription, this might include groups who speak a regional dialect or people who are a  
+native or a non-native speaker.
+
+It is also important to consider group intersections, for example, in addition
```

Comment:
I'd love to see a reference to Kimberle Crenshaw's work on intersectionality here. Wdyt?

Reply:
Sure! I added a sentence _Kimberlé Crenshaw's `seminal work on intersectionality <https://www.jstor.org/stable/1229039>`_ 
offers a thorough background on this topic._, which links to the JSTOR file, in lieu of creating a reference at the bottom of the page. I don't see this paper in the refs.bib file though, and I think we should add it there because it's referenced in multiple places!

Reply:
We only recently moved to BibTex which would explain the absence of the reference. I think it would be great if you could add it as part of this PR! You could also possibly adjust the references on the other places in the user guide you've spotted - but we can also leave that to another PR.

Reply:
Ah, gotcha. I'll update the .bib file with the Crenshaw paper and add it to this PR. I do think there's room for a larger task of adding all the referenced links throughout the user guide to that file. I think this is a good first issue, so I can open up that PR for another contributor to work on!

Reply:
Agreed! Great suggestion to open an issue for this!